debian/rules

```diff
@@ -29,6 +29,7 @@ include /usr/share/dpkg/default.mk
 
 override_dh_auto_configure:
 	dh_auto_configure --
+	dh_auto_configure -- --enable-gtest
```

Comment:
>--enable-gtest [](start = 22, length = 14)

It will not work in sonic-slave. The tests require redis service running.

Reply:
Not sure how sonic prepares unit test environment in Jenkins build.    I install the debian package in sonic-slave and start redis service with "redis-server  &" commond.

Reply:
You got it clear. So just keep original version, and manually test it with
```
./configure --enable-gtest
make
```